newitems.asm

```diff
@@ -1118,7 +1118,7 @@ RTL
 ; Set a flag in SRAM if we pick up a compass in its own dungeon with HUD compass
 ; counts on
 MaybeFlagCompassTotalPickup:
-        LDA CompassMode : BEQ .done
+        LDA.l CompassMode : AND.w #$000F : BEQ .done
```

Comment:
This was causing a crash. Needs to be 'AND.b #$0F' instead because this is called when the m flag is on.